knockout-tinymce.coffee

```diff
@@ -7,7 +7,6 @@
     defaults: {}
     extensions: {}
     init: (element, valueAccessor, allBindings, viewModel, bindingContext) ->
-      throw "valueAccessor must be writeable and observable"  unless ko.isWriteableObservable(valueAccessor())
```

Comment:
that line is removed since after this change, the binding can write back to normal properties. Basically, even this should work:
`data-bind="tinymce: prop"`
`var vm = {}; ko.applyBindings(vm);`
and the vm would get a simple string property with the value from tinymce whenever it changes.